### ISSUE TITLE
How to configure JobStorage on the client side?

### ISSUE THREAD

**Issue by ilya-chumakov** [ahydrax/Hangfire.PostgreSql#6]
Could you give me some advice how to configure the database access on the client side to add jobs using `RecurringJob` and `BackgroundJob` classes without running the whole Hangfire server?

This is how it is done in `frankhommers/Hangfire.PostgreSql`:

            GlobalConfiguration
                .Configuration
                .UsePostgreSqlStorage(options.ConnectionString);

However, it seems that `UsePostgreSqlStorage` extension doesn't exist in this package.

**Comment by ahydrax** [comment]
The same extension method is implemented in my package (located in `PostgreSqlBootstrapperConfigurationExtensions` class). As an alternative you can use:
`GlobalConfiguration.Configuration.UseStorage(new PostgreSqlStorage(connectionString, options))`

**Comment by ilya-chumakov** [comment]
@ahydrax thanks. It seems that Intellisense just missed the right method after I had switched the packages last time. I have tried again with explicit `dotnet restore` and now it works.

FYI, this code threw an exception:
    
    Npgsql is not fully compatible with TransactionScope yet, only connections without Enlist = true are accepted.
I've tried to explicitly add `Enlist = true` to my connection string but it hasn't helped. Then I've changed the statement to `Enlist = false` and now Hangfire connects to the db successfully on both server and client sides. My current connection string looks like this:
    
    "Host=XXX;Port=5432;Database=scheduler;Username=scheduler;Password=XXX;Enlist=False;Pooling=False"
